var/spack/repos/builtin/packages/py-bluepyemodel/package.py

```diff
@@ -25,22 +24,15 @@ class PyBluepyemodel(PythonPackage):
     depends_on("py-pyyaml", type=("build", "run"))
     depends_on("py-gitpython", type=("build", "run"))
     depends_on("py-bluepyopt@1.12.12:", type=("build", "run"))
-    depends_on("py-bluepyefe", type=("build", "run"))
+    depends_on("py-bluepyefe@2.2.0:", type=("build", "run"))
     depends_on("py-neurom@3.0:3", type=("build", "run"))
     depends_on("py-efel@3.1:", type=("build", "run"))
     depends_on("py-configparser", type=("build", "run"))
-    depends_on("py-bluepy@2.4:", when="@0.0.8", type=("build", "run"))
     depends_on("py-morph-tool@2.8:", type=("build", "run"))
     depends_on("py-fasteners@0.16:", type=("build", "run"))
     depends_on("neuron+python@8.0:", type=("build", "run"))
     depends_on("py-jinja2@3.0.3", when="@0.0.11:", type=("build", "run"))
-    # missing but needed
-    depends_on("py-click@7.0:", when="@0.0.8", type=("build", "run"))
-    depends_on("py-matplotlib@2.2:", type=("build", "run"))
-    # extra
-    depends_on("py-bluepyparallel@0.0.3:", when="@0.0.8", type=("build", "run"))
-    depends_on("py-bglibpy@4.4:", type=("build", "run"))
-    depends_on("py-seaborn@0.11:", when="@0.0.8", type=("build", "run"))
+    depends_on("py-currentscape@0.0.11:", type=("build", "run"))
```

Comment:
also here the latest currentscape version on pypi is 0.1.1.dev12

Reply:
Is there any release planned for that? These `devX` versions don't play nice with Spack's definition of versions.

Reply:
Wait, we're talking about https://pypi.org/project/currentscape/#history , right?
Then the latest version is 1.0.12

There's a spack recipe in the bluebrain repo that should probably be open-sourced as well (aka moved to the builtin repo and filed for upstream merging), just like several other dependencies (open-sourced, but spack recipe still in bluebrain). I'll move all of those to the builtin repo (and bother people with questions when I have them) on this PR.

Once this PR eventually gets merged in I can take care of upstreaming as well.

Reply:
@AurelienJaquier but if I put a colon after '0.0.11', won't it just install the newest version that fits? I'm using the same requirement as in our setup.py

Reply:
It should work like that, yes. I have an impression that upstream generally likes things to be as similar to setup.py as possible, so it might be better to leave it like this then.